### ISSUE TITLE
R3: interesting but more analysis of non-Asian ethnicities helpful

### ISSUE THREAD

**Issue by cgreene** [greenelab/iscb-diversity-manuscript#48]
> This is an important reflective effort to undertake, and I am glad that it was performed.  The analysis on the whole seems reasonable.  It would be good to see more citations of this type of work in other disciplines and a comparison of the results here to what has been found in other scientific organizations. It also seems that the work could have been a bit more statistically rigorous in spots, and it would be nice to see more analysis of non-Asian ethnicities.

**Comment by trangdata** [comment]
> It also seems that the work could have been a bit more statistically rigorous in spots

I think this was addressed as we addressed #63 and #55 (in [here](https://github.com/greenelab/iscb-diversity-manuscript/pull/83) and [here](https://github.com/greenelab/iscb-diversity-manuscript/pull/84) respectively).

> It would be good to see more citations of this type of work in other disciplines and a comparison of the results here to what has been found in other scientific organizations.

We have included all relevant literature we could find on the topic. Many papers on academic diversity of various disciplines, as we discussed in the Discussion section, focus on the underrepresentation of women in different academic settings (e.g. invited by journals to submit papers less often, suggested as reviewers less often, or cited less often), but only a few focus on race (e.g. proposals by Asian, black or African-American applicants were less likely to be funded).

> it would be nice to see more analysis of non-Asian ethnicities

We did break down the analysis for other groups in one [analysis notebook](https://greenelab.github.io/iscb-diversity/10.predict-race.html#sup_fig_s2), but it is difficult to come to a definite conclusion due to the small sample size. However, our new affiliation analysis at the country level helps complement what we did.
